.babelrc

```diff
@@ -1,6 +1,8 @@
 {
   "presets": ["es2015", "stage-1", "react"],
   "plugins": [
+    "flow-react-proptypes",
+    "transform-flow-strip-types",
```

Comment:
I believe that _neither_ of these are included in the [react preset](https://github.com/babel/babel/blob/master/packages/babel-preset-react/index.js)